superflore/generators/bitbake/ros_meta.py

```diff
@@ -45,8 +45,11 @@ def commit_changes(self, distro, commit_msg):
             'files/{0}/newer-platform-components.list'.format(distro))
         self.repo.git.add(
             'files/{0}/superflore-change-summary.txt'.format(distro))
-        info('Committing to branch {0}...'.format(self.branch_name))
-        self.repo.git.commit(m=commit_msg)
+        if self.repo.git.status('--porcelain') == '':
```

Comment:
Could you please add this behavior to the Gentoo generator as well?

Reply:
Sure, but I think it should appear in a separate commit.

Reply:
The new issue is #245.